x-pack/plugins/observability_solution/slo/public/pages/slos/components/card_view/slo_card_item_actions.tsx

```diff
@@ -50,10 +25,20 @@ interface Props {
 }
 
 export function SloCardItemActions(props: Props) {
-  const euiShadow = useEuiShadow('l');
+  const borderRadius = useEuiTheme().euiTheme.border.radius.medium;
 
   return (
-    <Container boxShadow={euiShadow} position={'relative'}>
+    <div
+      css={css`
+        display: inline-block;
+        position: relative;
+        bottom: 42px;
+        left: 12px;
+        z-index: 1;
+        border-radius: ${borderRadius};
+        ${useEuiShadow('l')}
+      `}
+    >
```

Comment:
[optional] You can skip line 28 by using the Emotion's `css` theme callback:

```suggestion
  return (
    <div
      css={({ euiTheme }) => css`
        display: inline-block;
        position: relative;
        bottom: ${euiTheme.size.xxl};
        left: ${euiTheme.size.m};
        z-index: 1;
        border-radius: ${euiTheme.border.radius.medium};
        ${useEuiShadow('l')}
      `}
    >
```